### ISSUE TITLE
Join a relationship not working with enableAjaxTable

### ISSUE THREAD

**Issue by aliromero** [Laravel-Backpack/CRUD#1019]
Hi,
in my model i have added 2 joins :
my model scope : 

```
public function scopeRep($query)
    {
        return $query
            ->join('shops','shops.id' , '=' ,'products.shop_id')
            ->join('shop_user','shop_user.shop_id' , '=' ,'shops.id')
            ->where('shop_user.user_id',Auth::user()->id)
            ->select('products.*');


    }
```

I did it by adding $this->crud->addClause('rep') to my controller. 

scope worked with dd(Product::rep()->get()) . but not work in ajax crud

and error in ajax console : 

> SQLSTATE[23000]: Integrity constraint violation: 1052 Column 'name' in field list is ambiguous (SQL: select `name` as `name`, `shop_id` as `shop_id`, `discount` as `discount`, `discount_exp` as `discount_exp`, `price` as `price`, `status` as `status`, `id` as `id` from `products` inner join `shops` on `shops`.`id` = `products`.`shop_id` inner join `shop_user` on `shop_user`.`shop_id` = `shops`.`id` where `shop_user`.`user_id` = 1 limit 25 offset 0)

**Comment by aliromero** [comment]
When disabling AjaxTable it's ok , no solution?

**Comment by tabacitu** [comment]
Hi @aliromero ,

I'm afraid it's because of a package we used for the ajax datatables. It's been a plague, raising all sorts of issues. In the next version (3.3) we've eliminated it, so it should work perfectly as we only use Eloquent. If everything goes well, I expect to launch 3.3 next week. If it's ok with you, use the regular table for a week, then upgrade to 3.3. Then all your tables will become ajaxdatatables, without this issue.

Cheers!
